package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "@daostack/test-env",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "Testing environment for DAOstack projects.",
   "main": "index.js",
   "scripts": {
-    "migrate": "node ./migrate.js",
     "release": "./release.sh",
-    "prepare-subgraph": "npm explore @daostack/subgraph -- npm i && node setup-subgraph.js",
-    "deploy-subgraph": "npm run prepare-subgraph && npm explore @daostack/subgraph -- npm run deploy",
-    "reset-migration": "cp -f node_modules/@daostack/migration/migration.json ./migration.json"
+    "deploy-ethereum": "node ./deployEthereum.js",
+    "deploy-subgraph": "node ./deploySubgraph.js",
+    "xxxreset-migration": "cp -f node_modules/@daostack/migration/migration.json ./migration.json"
```

Comment:
What's that for?

Reply:
you wrote it, I just added the xxx's to mark is as "why would you want  to do that?" 
removed it :-)